docs/development.md

```diff
@@ -0,0 +1,31 @@
+# Development Guide
+
+This document explains how to develop cluster-api-provide-openstack.
+
+## Using your own openstack-cluster-api-controller image for testing
```

Comment:
Please add some words explaining why you need to create your own image, what the image is used for, how it needs to be stored, etc...

Reply:
Sure, I will add.